app/src/full/java/io/homeassistant/companion/android/sensors/HealthConnectSensorManager.kt

```diff
@@ -0,0 +1,168 @@
+package io.homeassistant.companion.android.sensors
+
+import android.content.Context
+import android.os.Build
+import android.os.Build.VERSION.SDK_INT
+import androidx.health.connect.client.HealthConnectClient
+import androidx.health.connect.client.permission.HealthPermission
+import androidx.health.connect.client.records.ActiveCaloriesBurnedRecord
+import androidx.health.connect.client.records.TotalCaloriesBurnedRecord
+import androidx.health.connect.client.records.WeightRecord
+import androidx.health.connect.client.request.AggregateRequest
+import androidx.health.connect.client.request.ReadRecordsRequest
+import androidx.health.connect.client.time.TimeRangeFilter
+import io.homeassistant.companion.android.common.R as commonR
+import io.homeassistant.companion.android.common.sensors.SensorManager
+import java.time.Instant
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+import java.time.ZoneOffset
+import java.time.temporal.ChronoUnit
+import kotlinx.coroutines.runBlocking
+
+class HealthConnectSensorManager : SensorManager {
+    companion object {
+        val activeCaloriesBurned = SensorManager.BasicSensor(
+            id = "health_connect_active_calories_burned",
+            type = "sensor",
+            commonR.string.basic_sensor_name_active_calories_burned,
+            commonR.string.sensor_description_active_calories_burned,
+            "mdi:fire",
+            unitOfMeasurement = "kcal"
```

Comment:
Should we consider adding a state class and entity category for these new sensors? I think some may apply?

Reply:
Added state class and entity category to all sensors